autoscale-docker-swarm-container/README.md

```diff
@@ -0,0 +1,167 @@
+    Used to specify the RabbitMQ Management API Endpoint.  Alternatively `CAF_RABBITMQ_HOST` and `CAF_RABBITMQ_MGMT_PORT` may instead be specified individually.
+
+ - `CAF_RABBITMQ_MGMT_USERNAME`  
+    Default: `guest`  
+    Used to specify the username used to connect to RabbitMQ.  If `CAF_RABBITMQ_MGMT_USERNAME` is not specified then `CAF_RABBITMQ_USERNAME` will also be checked before falling back to the default.
+
+ - `CAF_RABBITMQ_MGMT_PASSWORD`  
+    Default: `guest`  
+    Used to specify the password used to connect to RabbitMQ.  If `CAF_RABBITMQ_MGMT_PASSWORD` is not specified then `CAF_RABBITMQ_PASSWORD` will also be checked before falling back to the default.
+
+ - `CAF_AUTOSCALER_DOCKER_SWARM_STACK`  
+    Required property used to specify the docker stack(s) which the autoscaler is to monitor.  Only applications deployed directly or indirectly within these stacks are auto-scaled.
+    To specify multiple stacks use a comma delimted list e.g. "JobService,DataProcessing".
+
+ - `CAF_AUTOSCALER_SCALING_DELAY`  
+    Minimum: `1`  
+    Default: `10`  
+    Used to specify the number of queue statistics records that must be acquired sequentially before performing analysis upon them and issuing a scaling recommendation. This means the period in time between making scaling recommendations is the service interval multiplied by the scalingDelay. This number should be small enough to respond to varying load but big enough to avoid erratic behaviour.
+
+ - `CAF_AUTOSCALER_BACKLOG_GOAL`  
+    Default: `300`  
+    Used to specify the amount of time, in seconds, that we ideally want to complete the current backlog of messages in. This is effectively a quality of service parameter, where lower will trigger more aggressive scaling.
+
+ - `CAF_AUTOSCALER_MAXIMUM_INSTANCES`  
+    Default: `100`  
+    Used to specify the maximum number of instances that any worker can be scaled to.
+
+ - `CAF_DOCKER_SWARM_TIMEOUT`  
+    Default: `30`  
+    Used to specify the max length of time in seconds that a docker REST call can take before a timeout out occurs.  
+
+ -  `CAF_DOCKER_SWARM_HEALTHCHECK_TIMEOUT`  
+    Default: `5`  
+    Used to specify the max length of time in seconds that the Docker endpoint healthcheck can take before a timeout occurs.
+
+ - `HTTP_PROXY`  
+    (Optional)
+    Used to specify an HTTP based proxy, for the Docker REST endpoint communication. 
+
+### Health checks
+
+The [autoscale-core](https://github.com/Autoscaler/autoscaler/tree/develop/autoscale-core) application inherently exposes standard and
+module-specific health checks. If you expose the admin port (default 8081)
+then this can be accessed via HTTP to examine metrics and health checks.
+The health check REST call will return HTTP 500 if any health check fails.
+For details on the health checks for specific components, examine the module
+documentation.
+
+
+### Logging
+
+Currently the logging is accessible via the Marathon sandbox until a CAF
```

Comment:
Debug comment is above whether to change both.
How much of the logging mandate should be brought across into this documentation.
As its not open sourced I was wondering is it going to be duplicated for every project, or is there somewhere we can link to for common logging documentation.  Say by placing it under caf-common or something?